primed/templates/cdsa/cdsaworkspace_detail.html

```diff
@@ -80,5 +81,23 @@ <h2 class="accordion-header" id="headingAcknowledgments">
   </div>
 </div>
 
+<div class="my-3">
+  <div class="accordion" id="accordionDataPrepWorkspace">
+    <div class="accordion-item">
+      <h2 class="accordion-header" id="headingDataPrepWorkspace">
+        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseDataPrepWorkspace" aria-expanded="false" aria-controls="collapseDataPrepWorkspace">
+          <span class="fa-solid fa-computer mx-2"></span>
+          Associated data prep workspace
```

Comment:
Can you make this show a badge indicating the number of associated data prep workspaces? It should look similar to the "View groups that this workspace is shared with" accordion from the ACM workspace detail page.